documentation/modules/ROOT/pages/operations/debezium-ui.adoc

```diff
@@ -23,7 +23,35 @@ The UI is implemented as a https://quarkus.io/[Quarkus]-based web application.
 
 == Installation and Configuration
 
-==== Debezium UI container image
+=== Debezium environment vars
+
+[cols="1,2,3,6",options="header"]
+
+|===
+|Item |ENV vars |application.properties vars |description
+
+|1
+|`UI_BASE_URI`
+|ui.base.uri
+|http://localhost:8080/api
+
+|2
+|`UI_MODE`
+|`ui.mode`
+|production
+
+|3
+|`DEPLOYMENT_MODE`
+|`deployment.mode`
+|validation.disabled
```

Comment:
I don't think this should be the default?

Reply:
Indeed. The default option is called `default`